### DISCUSSION TITLE
OclipResNet实现

### DISCUSSION THREAD

**Discussion by w1125** [open-mmlab/mmocr#1856] [General]
查看CLIPResNet代码，没有发现原代码中的 AttentionPool2d实现
self.attnpool = AttentionPool2d(input_resolution // 32, embed_dim, heads, output_dim)
x, att_maps = self.attnpool(x)
请问在mmocr中的CLIPResNet是否没有加入注意力，那么实验结果与预训练模型是否对应？
如果有实现过程，希望能得到实现代码的位置。
感谢回复。

**Comment by gaotongxiao** [comment]
It's implemented based on the OCLIP ResNet variant instead of the original CLIP ResNet, and therefore the architecture might be different.
